### DISCUSSION TITLE
Wifi Icon doesn't work despite installed hack nerd font

### DISCUSSION THREAD

**Discussion by alvaroaleman** [FelixKratz/SketchyBar#389] [Q&A] [answered]
Hi there,

I am using the [wifi config from the default sketchybarrc](https://github.com/FelixKratz/SketchyBar/blob/e67b96aa7dc7954b6f430ebcd24e593f913d6bb9/sketchybarrc#L94) and followed the [guide to install the Hack Nerd Font](https://felixkratz.github.io/SketchyBar/setup#fonts). Despite that, the icon doesn't seem to work, see the screenshot.

This is on Ventura. Any idea what the reason could be?

Thanks for any help in advance!

<img width="27" alt="image" src="https://github.com/FelixKratz/SketchyBar/assets/6496100/276f14d0-1021-4aaf-9eea-a9e9152ce1cb">

**Comment by FelixKratz** [comment, marked as answer]
It maybe has to do with the nerdfont update to v3. I have tried to replace the icon here: https://github.com/FelixKratz/SketchyBar/commit/672fe102273585dace21bc62b041633883c8e365 could you try the new one?

**Reply by alvaroaleman** [reply]
Yes, the new one worked. Thanks for taking the time to look into it!
